spec/Cabin/Infotainment.vspec

```diff
@@ -211,3 +211,20 @@ SmartphoneProjection.SupportedMode:
   type: attribute
   allowed: [ 'ANDROID_AUTO', 'APPLE_CARPLAY', 'MIRROR_LINK', 'OTHER' ]
   description: Supportable list for projection.
+
+PowerOptimize:
+  type: branch
+  description: All PowerOptimize actions.
+
+PowerOptimize.DisplayBrightnessLevel:
+  datatype: float
+  type: actuator
+  unit: percent
+  min: 0
+  max: 100
+  description: Display BrightnessLevel in the case of PowerOptimize
```

Comment:
Discussed in meeting. Intended usage is rather:

description: "Brightness level proposed by power optimization system. It is up to HMI to decide if it want to follow advice."